context.go

```diff
@@ -282,11 +282,16 @@ func (c *context) RealIP() string {
 	if ip := c.request.Header.Get(HeaderXForwardedFor); ip != "" {
 		i := strings.IndexAny(ip, ",")
 		if i > 0 {
-			return strings.TrimSpace(ip[:i])
```

Comment:
trims are ok but we are missing now `strings.TrimSpace`

Reply:
Oops, sorry, fixed now.